app/models/host.rb

```diff
@@ -908,7 +908,7 @@ def reset_discoverable_fields
   def detect_discovered_os(ost)
     # Determine os
     os_type = nil
-    if vmm_vendor == "vmware"
+    if vmm_vendor_display == "vmware" # ??? This will never be true
```

Comment:
detect_discovered_os, I believe, is for bare-metal Host provisioning.  @gmcculloug @bdunne Can you comment here?